client/src/client.ts

```diff
@@ -69,22 +79,54 @@ export class AngularLanguageClient implements vscode.Disposable {
         provideHover: async (
             document: vscode.TextDocument, position: vscode.Position,
             token: vscode.CancellationToken, next: lsp.ProvideHoverSignature) => {
-          if (isInsideInlineTemplateRegion(document, position)) {
-            return next(document, position, token);
+          if (!isInsideInlineTemplateRegion(document, position)) {
+            return;
           }
+
+          const angularResultsPromise = next(document, position, token);
+
+          const vdocUri = this.createVirtualHtmlDocForPosition(document, position);
+          const htmlProviderResultsPromise = vscode.commands.executeCommand<vscode.Hover[]>(
+              'vscode.executeHoverProvider', vdocUri, position);
+
+          const [angularResults, htmlProviderResults] =
+              await Promise.all([angularResultsPromise, htmlProviderResultsPromise]);
+          return angularResults ?? htmlProviderResults?.[0];
         },
         provideCompletionItem: async (
             document: vscode.TextDocument, position: vscode.Position,
             context: vscode.CompletionContext, token: vscode.CancellationToken,
             next: lsp.ProvideCompletionItemsSignature) => {
-          if (isInsideInlineTemplateRegion(document, position)) {
-            return next(document, position, context, token);
+          // If not in inline template, do not perform request forwarding
+          if (!isInsideInlineTemplateRegion(document, position)) {
+            return;
           }
+          const angularCompletions = next(document, position, context, token) as
+              Promise<vscode.CompletionItem[]|null|undefined>;
+
+          const vdocUri = this.createVirtualHtmlDocForPosition(document, position);
+          // This will not include angular stuff because the vdoc is not associated with an angular
+          // component
+          const htmlProviderCompletions = vscode.commands.executeCommand<vscode.CompletionList>(
+              'vscode.executeCompletionItemProvider', vdocUri, position, context.triggerCharacter);
+          const results = await Promise.all([angularCompletions, htmlProviderCompletions]);
+
+          return [...(results[0] ?? []), ...(results[1]?.items ?? [])];
```

Comment:
nit: Consider using the destructing pattern on line 92. I think that's more readable.